enterprise_subsidy/apps/content_metadata/api.py

```diff
@@ -70,7 +70,7 @@ def product_source_for_content(self, content_data):
         Helps get the product source string, given a dict of ``content_data``.
         """
         if product_source := content_data.get('product_source'):
-            source_name = product_source.get('name')
+            source_name = product_source.get('slug')
```

Comment:
Prefer using the slug over name, as name is usually something that can change at any time.